src/documents/forms/EditDocumentForm.js

```diff
@@ -49,7 +50,16 @@ const EditDocumentForm = React.forwardRef(
             file_desc: fileDescription,
             file_status: versionStatus,
           }}
-          validate={vals => validate(vals, fileNode, studyFiles)}
+          validate={vals => {
+            setTitleError(validate(vals, fileNode, studyFiles));
+            let errors = {};
+            ['file_name', 'file_type', 'file_desc'].forEach(function(field) {
```

Comment:
would `Object.keys(vals)`work here ?

Reply:
No, because we don't want to have `file_status` here since that's only for editing existing file.